src/Compilers/Core/Portable/Operations/IArgument.cs

```diff
@@ -24,13 +24,13 @@ public interface IArgument : IOperation
         /// </summary>
         IOperation Value { get; }
         /// <summary>
-        /// Conversion applied to the argument value passing it into the target method. Applicable only to VB Reference arguments.
+        /// Information of the conversion applied to the argument value passing it into the target method. Applicable only to VB Reference arguments.
         /// </summary>
-        IOperation InConversion { get; }
+        Optional<CommonConversion> InConversion { get; }
```

Comment:
>Optional<CommonConversion> [](start = 8, length = 26)

It doesn't look like it is a common pattern to use ```Optional<T>``` in IOperation API. Would it be better to use identity conversion for situations when no conversion is needed?